lib/src/main/java/my/nanihadesuka/compose/LazyColumnScrollbar.kt

```diff
@@ -226,8 +227,8 @@ fun InternalLazyColumnScrollbar(
             val offset = realFirstVisibleItem
                 ?.size
                 ?.let { it.toFloat() * remainder }
-                ?.toInt() ?: 0
-            listState.scrollToItem(index = index, scrollOffset = offset)
+                ?: 0f
+            listState.scrollBy(offset)
```

Comment:
Although it works, there is a good reason not to use relative displacement (scrollBy) but instead use absolute displacement (scrollToItem). When the user holds the scrollbar and moves it the "micro-displacements" could cause instability (numerical) if done by relative displacement as the previous calculations might not have been done (recomposition) or skipped as a new one are created (remember we are using a coroutine). Better to have deterministic behaviour.

Note that this doesn't show in the UI unit tests (I don't know how to reproduce programmatically).